frontend/src/lib/components/Feedback/FeedbackForm.svelte

```diff
@@ -0,0 +1,109 @@
+<script lang="ts">
+    import '$styles/Feedback.pcss';
+    import '$styles/Auth.pcss';
+    import EmailIcon from '$icons/Email.svelte';
+    import IncrementIcon from '$icons/Increment.svelte';
+    import DecrementIcon from '$icons/Decrement.svelte';
+    import PersonIcon from '$icons/Person.svelte';
+    import '$styles/Button.pcss';
+
+    let name: string = "";
+    let email: string = "";
+    let chapter: number = 0;
+    let question: number = 0;
+    let feedback: string = "";
+
+    const submitFeedback = () => {
+        fetch('https://script.google.com/a/macros/umass.edu/s/AKfycbyy6_ndlSZu_VUC_tdgoQhoOsxyUEBIKZ1YvwJGnwJSy_DLBqDr2uYiiUkVot-fHWRQ/exec', {
```

Comment:
Can you explain of how this fetching works to the google script server?

Reply:
We might need to use svelte kit action (server side) to send a request to the server. You can read more about that from here: https://vercel.com/guides/using-sveltekit-form-actions